src/provider.c

```diff
@@ -544,7 +544,11 @@ static int p11prov_module_init(P11PROV_CTX *ctx)
 
     dlerror();
     ctx->dlhandle = dlopen(ctx->module,
-                           RTLD_NOW | RTLD_LOCAL | RTLD_DEEPBIND);
+                           RTLD_NOW | RTLD_LOCAL
+#if defined(RTLD_DEEPBIND)
+			   | RTLD_DEEPBIND
+#endif /* defined(RTLD_DEEPBIND) */
+	    );
```

Comment:
Uuuh this is too ugly,
what about adding at the top of the file:

#if !defined(RTLD_DEEPBIND)
#define RTLD_DEEPBIND 0
#endif

this will deuglify usage later.

Reply:
Sure, works for me.